### ISSUE TITLE
Negative Hover Height

### ISSUE THREAD

**Issue by UniDuki** [studio-minus/ppg-bugs-and-requests#116]
**TLDR: Blue signals bypass the minimum hover height**

**Describe the bug**
Hover thrusters don't like having a negative hover height value, you can't set it below 0 manually, but using blue wires you can.

**Clearly describe how to reproduce the bug**
Send a blue signal to a hover thruster until it's hover height is below 0, then activate it.

For me it, and anything connected to it, will jump to a specific place in the floor* and constantly try to phase downwards but keeps getting teleported back up. 

**Additional notes**
Opening and closing the set hover height menu will set the height back up to 0.5m, even if you cancel and don't click apply.

This may be what https://github.com/studio-minus/ppg-bugs-and-requests/issues/115 is trying to say

# EXPORTED GAME DEBUG INFO
## System info

 - **OS**: Windows 10  (10.0.19043) 64bit

 - **CPU**: Intel(R) Core(TM) i3-1005G1 CPU @ 1.20GHz

 - **GPU**: Intel(R) UHD Graphics

 - **Memory**: 7 GB

## All loaded mods
None

*Bottom left corner of default map
![image](https://user-images.githubusercontent.com/69184577/130330708-6681f62d-504a-4dfa-8ed4-d5c0f0d7da86.png)
![image](https://user-images.githubusercontent.com/69184577/130330737-3f651c39-b5e1-4a49-8a61-f0a068ebff15.png)

**Comment by UniDuki** [comment]
This also works the other way round, where you can go over the max hover height.

**Comment by mestiez** [comment]
Oh no
